import-accession.sh

```diff
@@ -32,8 +32,7 @@ for ACCESSION in "$@"; do
         echo "$SRA_FILES" | awk '{print $NF}' | tr ';' '\n' | xargs -P 4 -I {} ./import-accession-single.sh $ACCESSION {}
 
         # Create metadata.tsv with SRA run names
-        echo "Run" > "${metadata_dir}/metadata.tsv"
-        echo "$SRA_FILES" | awk '{print $1}' >> "${metadata_dir}/metadata.tsv"
+        echo "$SRA_FILES" | awk -F'[/.]' '{print $(NF-2)}' >> "${metadata_dir}/metadata.tsv"
```

Comment:
`>` would be better than `>>` so we don't end up with extra lines when running multiple times